packages/client/src/config.ts

```diff
@@ -93,7 +93,10 @@ export function createBundleBuilder(
     case 'messageSignature':
       return new MessageSignatureBundleBuilder(bundlerOptions);
     case 'dsseEnvelope':
-      return new DSSEBundleBuilder(bundlerOptions);
+      return new DSSEBundleBuilder({
+        ...bundlerOptions,
+        certificateChain: true,
+      });
```

Comment:
Will continue to generate v0.2 bundles by default in the `client` package (for now).